fun will_throw
{
  var  exc = void
try {
  [ will_throw()... ]
""}
catch const e { e
}

try {
[ will_throw()...]
}
catch const e {

return   exc-> e.value
}
8
}
var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
}
assert exc
